apps/src/templates/Button.jsx

```diff
@@ -25,13 +25,15 @@ const ButtonColor = {
 const ButtonSize = {
   default: 'default',
   large: 'large',
-  narrow: 'narrow'
+  narrow: 'narrow',
+  small: 'small'
```

Comment:
This new `small` size also has a `top: 4px` offset to make it align better with surrounding text. I'm open to other options if that side-effect is too weird.